.github/workflows/publish.yml

```
@@ -1,29 +1,86 @@
-name: brew pr-pull
+name: Publish Bottles
 on:
-  pull_request_target:
-    types:
-      - labeled
+  pull_request:
+    paths:
+      - Formula/*.rb
+
 jobs:
-  pr-pull:
-    if: contains(github.event.pull_request.labels.*.name, 'pr-pull')
-    runs-on: ubuntu-22.04
+  build-bottles:
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest]
+    runs-on: ${{ matrix.os }}
     steps:
       - name: Set up Homebrew
         uses: Homebrew/actions/setup-homebrew@master
 
+      - run: brew test-bot --only-cleanup-before
+
+      - run: brew test-bot --only-setup
+
+      - run: brew test-bot --only-tap-syntax
+
+      - run: brew test-bot --only-formulae
+
+      - name: Upload bottles as artifact
+        uses: actions/upload-artifact@main
```

```suggestion
        uses: actions/upload-artifact@v3
```
...and then also setup dependabot for updating actions.